api/config/cdr_config_local.json

```diff
@@ -7,6 +7,14 @@
       "servicePerimeter": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test",
       "authDomainName": "all-of-us-registered-test",
       "authDomainGroupEmail": "GROUP_all-of-us-registered-test@dev.test.firecloud.org"
+    },
+    {
+      "accessTierId": 2,
+      "shortName": "controlled_test",
+      "displayName": "Fake Controlled Tier For Test/Local",
+      "servicePerimeter": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test_2",
+      "authDomainName": "all-of-us-test-prototype-3",
+      "authDomainGroupEmail": "all-of-us-test-prototype-3@dev.test.firecloud.org"
```

Comment:
I saw another tier has  `GROUP_` as prefix. Does this one need that?

Reply:
no.  I believe this is because Terra has changed how they create group email addresses over time.  See perf, prod, and preprod configs for other examples of the new style.